test/e2e/apimachinery/chunking.go

```diff
@@ -122,6 +129,61 @@ var _ = SIGDescribe("Servers with support for API chunking", func() {
 		gomega.Expect(list.Items).To(gomega.HaveLen(numberOfTotalResources))
 	})
 
+	ginkgo.It("should chunk lists of Pods", func() {
```

Comment:
Why are we adding a new test instead of just updating the existing ones to use config maps?  What does this new test add?